articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md

```diff
@@ -59,23 +59,10 @@ Note the following information before you install the plug-in:
 
 The plug-in supports the following versions of Jira and Confluence:
 
-* Jira Core and Software: 6.0 to 7.12
-* Jira Service Desk: 3.0.0 to 3.5.0
+* Jira Core and Software: 6.0 to 8.22.1
+* Jira Service Desk: 3.0.0 to 4.22.1
 * JIRA also supports 5.2. For more details, click [Microsoft Azure Active Directory single sign-on for JIRA 5.2](./jira52microsoft-tutorial.md)
```

Comment:
You have missed to update the versions of JIRA and Confluence in the "Which versions of Jira and Confluence does the plug-in support?" section. Can you update that section with the JIRA and Confluence version?